design-system/react/src/components/Card/defs.ts

```diff
@@ -0,0 +1,57 @@
+import type { BoxProps, FlexProps } from '../Box';
+
+import type { CardBody } from './CardBody';
+import type { CardFooter } from './CardFooter';
+import type { CardHeader } from './CardHeader';
+
+import type { Components } from '~/defs';
+import type { CreateComponent } from '~/utils';
+
+export type CardProps = FlexProps & {
```

Comment:
In order to get `FlexProps` working as expected, you can create some kinda a hook `useFlexProps` that does the same inside the `Flex` component and use it alongside namespaced components and `Card`